examples/knowledge_tuning/knowledge_tuning_with_reasoning_model/reasoning_sdg.py

```
@@ -0,0 +1,1032 @@
+#
+# In `sdg_hub`, the seed data must adhere to a **strict column naming convention**:
+# Each column name must **exactly match** the variable names expected in the prompt template of the first `LLMBlock` in the flow.
+#
+# This alignment is crucial—every subsequent block in the pipeline relies on the output columns of the previous block to match its own input variable names. This ensures that prompts are populated correctly and that the flow remains **fully compatible and composable** from block to block.
+#
+# Maintaining this column-to-variable consistency is essential for smooth data propagation and accurate prompt construction across the entire synthetic data generation pipeline.
+#
+
+# %%
+# Create a simple dataset
+from datasets import load_dataset
```

Flake8 (7.2.0)

[error] redefinition of unused 'load_dataset' from line 45

(F811)